Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:8
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+
+RUN yarn install
```

Comment:
Do also need to run `yarn build` at some point?

Reply:
build is handled here:
https://github.com/hmcts/sscs-cor-frontend/pull/69/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12
to be inline with local approach